packages/mobile/src/account/RaiseLimitScreen.tsx

```diff
@@ -0,0 +1,211 @@
+import Button, { BtnSizes, BtnTypes } from '@celo/react-components/components/Button'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import React, { useEffect, useMemo } from 'react'
+import { useAsync } from 'react-async-hook'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useDispatch } from 'react-redux'
+import { updateDailyLimitRequestStatus } from 'src/account/actions'
+import { sendEmail } from 'src/account/emailSender'
+import { DailyLimitRequestStatus } from 'src/account/reducer'
+import { cUsdDailyLimitSelector, dailyLimitRequestStatusSelector } from 'src/account/selectors'
+import { showMessage } from 'src/alert/actions'
+import { CELO_SUPPORT_EMAIL_ADDRESS } from 'src/config'
+import { readOnceFromFirebase } from 'src/firebase/firebase'
+import i18n, { Namespaces } from 'src/i18n'
+import ApprovedIcon from 'src/icons/ApprovedIcon'
+import DeniedIcon from 'src/icons/DeniedIcon'
+import InProgressIcon from 'src/icons/InProgressIcon'
+import { headerWithBackButton } from 'src/navigator/Headers'
+import { navigate, navigateBack } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import useSelector from 'src/redux/useSelector'
+import Logger from 'src/utils/Logger'
+import { currentAccountSelector } from 'src/web3/selectors'
+
+const UNLIMITED_THRESHOLD = 99999999
+
+const RaiseLimitScreen = () => {
+  const { t } = useTranslation(Namespaces.accountScreen10)
+  const dailyLimit = useSelector(cUsdDailyLimitSelector)
+  const dailyLimitRequestStatus = useSelector(dailyLimitRequestStatusSelector)
+  const numberIsVerified = useSelector((state) => state.app.numberVerified)
+  const address = useSelector(currentAccountSelector)
+  const dispatch = useDispatch()
+
+  const applicationStatusResult = useAsync(async () => {
+    if (!address) {
+      return null
+    }
+    return readOnceFromFirebase(`dailyLimitRequest/${address}`)
+  }, [])
```

Comment:
Will not change anything in this specific case, but just for the sake of correctness 🤓 
```suggestion
  }, [address])
```